### DISCUSSION TITLE
Runner is always on Queue

### DISCUSSION THREAD

**Discussion by kstoletniy** [community/community#83946] [Actions]
### Select Topic Area

Bug

### Body

I use Github Action to submit iOS builds to AppCenter. Last week was working properly, however today the workflow is not executed and is always in the state Queued. Nothing was changed in the workflow.

The runner is [self-hosted, macOS, ARM64, iOS]

Has anyone faced this issue today? Any idea of what’s going on?

**Comment by JesseTG** [comment]
This issue occurs for me as well. See [here](https://github.com/orgs/community/discussions/82805#discussioncomment-8082066) for more details and links to stalled runs.
